bigdata/src/main/scala/org/w3/banana/bigdata/Bigdata.scala

```diff
@@ -0,0 +1,30 @@
+package org.w3.banana.bigdata
```

Comment:
Ok, I will add it in the next commit. However Intellij IDEA highlights me double package definition as a "bad practise"